microsoftgraph/microsoft-graph-explorer-v2
# Disables editing on code snippets

## Overview

Disables editing on code snippets

### Demo

Optional. Screenshots, `curl` examples, etc.
![image](https://user-images.githubusercontent.com/12011447/67369272-66746080-f581-11e9-9a13-9ce7f567d6f5.png)

### Notes

N/A

## Testing Instructions

* Click on Snippets tab
* Try editing a snippet
* Observe that you can't edit

Fixes #190
